### ISSUE TITLE
qsub returns an error message

### ISSUE THREAD

**Issue by pditommaso** [gawbul/docker-sge#5]
I've launched the container like this: 

```
docker run --rm -it gawbul/docker-sge:latest bash
```

Then I've tried to submit a job: 

```
# echo 'echo Hello' | qsub
Unable to run job: unable to send message to qmaster using port 6444 on host "9818fcbe520e": can't resolve host name
Exiting.
```

**Comment by gawbul** [comment]
@pditommaso, thanks for submitting this. If you run with `docker run --rm -it gawbul/docker-sge:latest /sbin/my_init -- bash` it should overcome the hostname issue, however, I have noticed the SGE configuration doesn't like root submitting jobs, so I am just working on fixing that.

**Comment by gawbul** [comment]
Thinking forward though, I would like to think of a way to run the `my_init` system when the user only requests bash, as you did? Perhaps add to `/sbin/my_init` /etc/bashrc?

**Comment by pditommaso** [comment]
Not sure about that. As far as I know, when you run `docker run -ti <container> bash` it reads the `~/.bashrc` file.

**Comment by gawbul** [comment]
Fixed in b13ada95769ec0babbf4ce916611c0a46d97926b :) @pditommaso
